tests/conftest.py

```
@@ -43,3 +44,108 @@ async def close(self) -> None:
+    """
+    import urllib.request
+
+    # サーバーが起動しているか確認
+    try:
+        req = urllib.request.Request(  # noqa: S310
+            f"{sandbox_api_base}/health", method="GET"
+        )
+        with urllib.request.urlopen(req, timeout=2) as resp:  # noqa: S310
+            if resp.status == 200:
+                return {"profile": sandbox_profile, "api_base": sandbox_api_base}
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)